hr_timesheet_sheet_no_negative/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
License missing

Reply:
See https://github.com/OCA/maintainer-tools/commit/2ffdff1df7675fd6cab14e39e4ff5c6d46b9edb1#diff-2dc8828130dfbafb770bf7e7c4ae7557.

Reply:
Not copyright, license https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

Reply:
ah, sorry